bert_squeeze/utils/scorers/lm_scorer.py

```diff
@@ -0,0 +1,222 @@
+from collections import defaultdict
+from typing import Dict, List
+
+import evaluate
+import numpy as np
+import torch
+from tabulate import tabulate
+from transformers import AutoTokenizer
+
+
+class LMScorer(object):
+    """
+    Scorer for language modeling tasks
+    """
+
+    def __init__(self, tokenizer_name: str = None, do_mismatch: bool = True):
+        if tokenizer_name is not None:
+            self.tokenizer = AutoTokenizer.from_pretrained(tokenizer_name)
+        else:
+            self.tokenizer = None
+        self.do_mismatch = do_mismatch
+
+        self.losses = defaultdict(list)
+        self.metrics = defaultdict(list)
+        self.mismatches = []
+
+    @property
+    def ppl(self):
```

Comment:
is `ppl` stands for `perplexity` ? I would use the full name for clarity as the shorter version is not that shorter